dct_backend/webscraper.py

```diff
@@ -0,0 +1,111 @@
+import requests
+from bs4 import BeautifulSoup
+import dct
+
+# DCT import
+DCT = dct.DCT()
+
+# setup BeautifulSoup with URL
+req = requests.get("https://emoryatlanta.cafebonappetit.com/cafe/dobbs-common-table/")
+soup = BeautifulSoup(req.content, "html.parser")
+
+# Title of webpage
+title = soup.title
+# print(title.prettify()) #prettify() can be a useful beautifulsoup method
+title = title.get_text()
```

Comment:
This will run once when the script is imported, put this in a method and return the soup

Reply:
It has to be run each day. I would also have to pass the soup to each other method and it would be more work that it's worth.

Reply:
Exactly, if you want it to run each day it has to be put in a method. This isn't something we can set a cron job for. We lose the runtime memory if we do that. We are storing the data during runtime, which is what we need access to.

Reply:
Okay fine. It wasn't that bad. Hopefully I included everything that needs to be in this method.